.gitignore

```diff
@@ -75,3 +79,11 @@ venv36/
 future-manage.py
 misago-admin.py
 db.sqlite3
+
+# Local development files
+/manage.py
+/forum/
+/avatargallery/
+/static/
+/theme/
```

Comment:
`theme` and `cron` files should be removed from gitignore if they are also removed on deployment.